hicommonwealth/commonwealth
# Remove `offchain` from `Subscriptions` table and model

## Link to Issue
Closes: #4368 

## Description of Changes
- Created a migration which removes the offchain prefix from 2 columns in the `Subscriptions` table.
- Removed the prefix from the Sequelize `Subcriptions` model and all of it's references.
- Removed the offchain prefix from all raw queries as well.

## Test Plan
- Interact with subscriptions:
  - Create a thread -> Check the subscription for that thread appears in the notifications settings
  - Disable the thread subscription
  - Re-enable the thread subscription

## Deployment Plan

**WARNING: do not merge/deploy before #4479 is merged.**

## Other Considerations

- Migration takes 0.044 seconds on Frack.
